### ISSUE TITLE
A keypad page to insert hidden symbols

### ISSUE THREAD

**Issue by graphemecluster** [DesModder/DesModder#749]
Some symbols untypable by default like Greek letters and ± are highly valuable for use as variables. Others like overline $\overline{x}$ and vector notations $\vec{v}$ and $\overrightarrow{AB}$ have their use in labels. It would be great if there is a keyboard to input them without copy-and-pasting from an external $\LaTeX$ editor. (It seems that first of all the keypad has to be enabled at the label text input field.)

**Comment by graphemecluster** [comment]
(I know text-mode do help, but it‘s hard to find out all the available symbols with text-mode and I don‘t want to type in text)

**Comment by jared-hughes** [comment]
Good idea! MathQuill does support `\vec` and `\overline` as an easy input (e.g. try writing "\vec" then a space in http://mathquill.com), so it would be possible to trigger this situation from keypad.

For the Greek letters specifically: The "Custom MathQuill Config" plugin supports "More Greek Letters" if you enable it. `gamma`, `chi`, etc.

Text Mode wouldn't really help here because it doesn't support any of these symbols (which are LaTeX hacks).
